template.yml

```diff
@@ -176,7 +176,7 @@ Resources:
         FunctionName: !Sub ${ResourcePrefix}-apigw-custom-authorizer
         CodeUri: ./functions/ApiGwCustomAuthorizer/
         Handler: app.handler
-        Runtime: nodejs12.x
+        Runtime: nodejs16.x
```

Comment:
What testing has been done for this change?

Reply:
The only testing done was to build locally running `make setup` on node version `v16.17.1` I have assumed, since there is no version specified in `package.json` I have bumped the version up to make cfn lint happy.

The following error is from lint 
```
E2531 Deprecated runtime (nodejs12.x) specified. Updating disabled since 2022-12-14. Please consider updating to nodejs16.x
template.yml:141:3

E2531 Deprecated runtime (nodejs12.x) specified. Updating disabled since 2022-12-14. Please consider updating to nodejs16.x
template.yml:172:3
```

Happy to stick to  node v12 and add ignore flag to lint?

Reply:
I don't think that's sufficient for a major node version upgrade. It'd need to at least have some functional testing to ensure it still works as expected.

Reply:
I suggest removing the node version change from this PR.

Reply:
ok will do, Thanks Chris.